standalone/default.cfg

```diff
@@ -0,0 +1,393 @@
+
+[DAEMON]
+# set to 1 to run the daemon service within this process;
+# set to 0 if the daemon is run separately (using script joinmarketd.py)
+no_daemon = 1
+
+# Port on which daemon serves; note that communication still
+# occurs over this port even if no_daemon = 1
+daemon_port = 27183
+
+# Currently, running the daemon on a remote host is
+# *NOT* supported, so don't change this variable
+daemon_host = localhost
+
+# by default the client-daemon connection is plaintext, set to 'true' to use TLS;
+# for this, you need to have a valid (self-signed) certificate installed
+use_ssl = false
+
+[BLOCKCHAIN]
+# options: bitcoin-rpc, regtest, bitcoin-rpc-no-history, no-blockchain
+# When using bitcoin-rpc-no-history remember to increase the gap limit to scan for more addresses, try -g 5000
+# Use 'no-blockchain' to run the ob-watcher.py script in scripts/obwatch without current access
+# to Bitcoin Core; note that use of this option for any other purpose is currently unsupported.
+blockchain_source = bitcoin-rpc
+
+# options: signet, testnet, mainnet
+# Note: for regtest, use network = testnet
+network = mainnet
+
+rpc_host = localhost
+# default ports are 8332 for mainnet, 18443 for regtest, 18332 for testnet, 38332 for signet
+rpc_port =
+
+# Use either rpc_user / rpc_password pair or rpc_cookie_file.
+rpc_user = bitcoin
+rpc_password = password
+#rpc_cookie_file =
+
+# rpc_wallet_file is Bitcoin Core wallet which is used for address and
+# transaction monitoring (it is watchonly, no private keys are stored there).
+# It must be created manually if does not exist, see docs/USAGE.md for more
+# information.
+rpc_wallet_file =
```

Comment:
Yes.. that is correct. Do you suggest adding our default value to the config as well to make that more obvious?

Reply:
I'd say if we already have a custom config then yes, let's add our defaults here.

Reply:
Just to be clear - I did not change it as I think we might be able to get rid of the custom config in the future. The more we change now, the harder this will become. I might be wrong and providing the config is what would make life actually much easier in the first place.. but as of now, I'd really stay away from too much adaptions. 
However, the case for `rpc_wallet_file` is valid as we already adapt it.
Unfortunately we must stick with `jm_webui_default` here : /

Reply:
> providing the config is what would make life actually much easier in the first place

I think in general this is more likely to be the case. Then there's one source of truth for all of the configuration, instead of it being split up across all the integrations where we'd need to pass them via docker-compose variables. This doesn't apply for this specific setting as we do set that in the entrypoint script so it doesn't _really_ matter here.